Графика для всех страниц

OzzOn1 opened the pull request:
---
:mortar_board: [Графика для всех страниц](https://up.htmlacademy.ru/htmlcss/33/user/1957975/tasks/5)

:boom: https://htmlacademy-htmlcss.github.io/1957975-sedona-33/3/

---
:mortar_board: [Базовая стилизация всех страниц](https://up.htmlacademy.ru/htmlcss/33/user/1957975/tasks/6)

OzzOn1 committed:
Выполнил  Графику  и базовую стилизацию  для всех страниц

OzzOn1 committed:
Изменил  пути к фото  т.к они не появились в гите

OzzOn1 committed:
2я попытка загрузить фото на гит

OzzOn1 committed:
Добавил правильные пути к картинкам в css

Dima-Dim commented:
Почему в этом пуллриквесте появилась Базовая стилизация всех страниц? Он же для Графики и кроме графики в нём ничего быть не должно.
https://up.htmlacademy.ru/htmlcss/33/help/4.5
```Приступать к следующему заданию по тому же проекту не стоит, пока не будет принято предыдущее. Это нужно для того, чтобы не потерялся какой-то код при работе над несколькими заданиями одновременно (возможно, это будет происходить в разных ветках), а в пулреквестах не появились конфликты. Но если наставник принял задание с исправлениями, тогда можно легко переходить к новому заданию.```

Dima-Dim reviewed (changes requested):
Оставить только графику.

OzzOn1 committed:
Добавил графику для всех страниц

Убрал пока файл css.

OzzOn1 commented:
Я вроде бы удалил css, но после того как на сайте академии нажал обновить пулреквест. Мне сайт выдал опять ту версию сайта со стилями.

Dima-Dim reviewed (changes requested):
Вижу Vector-index-backgriund.svg и другие подобные названия, значит вопрос по критерию https://up.htmlacademy.ru/htmlcss/33/criteries#PROJ-03 всё ещё актуален.

OzzOn1 committed:
Изменил названия картинок

OzzOn1 committed:
Внес некоторые парвки

Добавил точки в  альты для скринридера что бы при  прочтении он  делал паузы. И пытаюсь  понять как вобще  работает  гит.

OzzOn1 committed:
Merge branch 'htmlacademy-htmlcss:master' into master

OzzOn1 committed:
Правка картинок

и название папки

OzzOn1 committed:
Merge branch 'master' of https://github.com/OzzOn1/1957975-sedona-33

Dima-Dim reviewed (approved):
images/vector-index-background.svg и images/vector_star.svg лучше использовать какой-то один подход, чтобы с этим было проще работать.

images/home-1.svg - https://up.htmlacademy.ru/htmlcss/33/criteries#PROJ-02
